assets/css/styles.css

```diff
@@ -399,7 +388,7 @@ body {
 }
 
 #form textarea {
```

Comment:
- Kindly give the inputs and textarea a uniform styling because currently, the form doesn't look like the Figma design

### Your Design
![contact-form6](https://user-images.githubusercontent.com/89091822/228617904-2a1cb58e-9804-4431-8f54-79f6eae7c61c.PNG)

### Figma Design
![figma-contact-form](https://user-images.githubusercontent.com/89091822/228618321-1b677b17-8b4a-43a9-b2f3-7e03723f699c.PNG)